jsapp/js/components/processing/singleProcessingStore.ts

```diff
@@ -105,7 +105,7 @@ interface SubmissionsEditIds {
   }>;
 }
 
-interface AutoTranscriptionEvent {
+interface AutoTransXEvent {
```

Comment:
@Guitlle  We already use `TransxObject` and `TransxLanguages` etc. (lowercase "x"). Let's keep this consistent

Reply:
Ok

Reply:
I've fixed that :ok: